test/test_pty.rs

```diff
@@ -0,0 +1,81 @@
+use nix::pty::openpty;
+use nix::sys::termios::*;
+use nix::unistd::{read, write, close};
+
+#[test]
+fn test_openpty() {
+    let pty = openpty(None, None).unwrap();
+    assert!(pty.master > 0); // must be valid file descriptors
```

Comment:
Please capitalize the "m" in "must" and move this comment to the above line if you want to keep it, though I don't see this comment as very useful. Maybe just remove these trailing comments from the code.